src/constants/index.ts

```diff
@@ -0,0 +1,2 @@
+export const SING_UP_URL = 'users';
+export const SING_IN_URL = 'sign_in';
```

Comment:
this should be on `src/constants/index.ts` there shouldnt be a index.ts on constants/utils global folders